tree/30_generic_methods/condition_from_expression_persistent.cf

```diff
@@ -18,16 +18,31 @@
 
 # @name Condition from expression persistent
 # @description Create a new condition class that persists across runs
-# @documentation This bundle will define a class `condition_from_expression_persistent_${condition_prefix}_{kept,ok,reached}`
-#
-# This bundle will additionally produce a `${condition_prefix}_true` or a `${condition_prefix}_false` condition depending on the result on the expression:
+# @documentation This method evaluates an expression, and produce a `${condition_prefix}_true` or a `${condition_prefix}_false` condition depending on the result on the expression, for the `duration` time:
 #
 # * If the expression results in a "defined" state, this will produce a kept outcome class and a `${condition_prefix}_true` condition
 # * If the expression results in an "undefined" state, this will produce a kept outcome class and a `${condition_prefix}_false` condition
 #
-# The created condition (class in cfengine speaking) is global to the agent and is persisted across runs.
+# Calling this method with a condition expression transforms a complex expression into a single class condition.
+#
+# The created condition is global to the agentand is persisted across runs.
```

Comment:
missing space

Reply:
could you also explain what persistence is, I don't think it is known by Rudder users.